apps/files/src/components/AllFilesList.vue

```diff
@@ -118,9 +118,17 @@ export default {
 
     $_emptyFolderText () {
       if (this.$_isFavoritesList) {
-        return this.$gettext('No files or folders marked as favorite')
+        return this.$gettext('There are no resources marked as favorite.')
       } else {
-        return this.$gettext('Folder is empty')
+        return this.$gettext('There are no resources in this folder.')
+      }
+    },
+
+    $_emptyFolderSubText () {
+      if (this.$_isFavoritesList) {
+        return this.$gettext('You can mark some by clicking on the star icon.')
```

Comment:
... `star icon in the file list`? As a new user I might not know where to find the star icon.